app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="loginpage"> 
+<div class="login" >
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] Users can register and sign-in correctly, however, I'm unable to sign out a user, and hence, I'm unable to test for other users. Let's ensure to improve upon this aspect.

![image](https://github.com/MayPyone/Recipe-app/assets/12745474/19855f1c-2c20-4e3e-871a-74a793c8398d)